flask_zipkin.py

```diff
@@ -117,7 +126,11 @@ def _after_request(self, response):
             return response
         if not hasattr(g, '_zipkin_span'):
             return response
-        g._zipkin_span.stop()
+        try:
+          g._zipkin_span.stop()
+        except Exception as e:
+          logging.warning('Unable to stop zipkin span:{}'.format(e))
```

Comment:
If the zipkin server crashes

Reply:
ok.. I suppose the record function (internal to span.stop()) should probably be explicit about it not being ok to block, or throw. This would obviate call sites from having to all do defensive code like this..

cc @drolando

Reply:
Yea it shouldn't ever crash